README.md

```diff
@@ -574,10 +574,8 @@ var userHandlerWrapper = func(m dsl.Message) error {
 
 // 3 Create the Pact Message Consumer
 pact := dsl.Pact {
-	return dsl.Pact{
-		Consumer:                 "PactGoMessageConsumer",
-		Provider:                 "PactGoMessageProvider",
-	}
+	Consumer: "PactGoMessageConsumer",
+	Provider: "PactGoMessageProvider",
```

Comment:
Hey @JosephWoodward ,

These above are taken from 

https://github.com/pact-foundation/pact-go/blob/a40972e9597b0112937c5e1af79308f253bfd9c9/examples/messages/consumer/message_pact_consumer_test.go#L110

also it is a func there rather than being assigned to a variable like here

https://github.com/pact-foundation/pact-go/blob/eb10d18029ed05894fc2c9319f93ea1dea961a89/examples/provider_test.go#L25

Reply:
You're right, but looks to be a bad copy/paste - in the source example, it was a `func` that returned an object, but in the copied example there is no func - so the suggestion is correct.